src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -1501,9 +1502,11 @@ public void onRemoteOperationFinish(RemoteOperation operation, RemoteOperationRe
     private void onRemoveFileOperationFinish(RemoveFileOperation operation,
                                              RemoteOperationResult result) {
 
-        showSnackMessage(
-                ErrorMessageAdapter.getResultMessage(result, operation, getResources())
-        );
+        if(getListOfFilesFragment().getNoOfItems() == 2 || result.isException()) {
```

Comment:
Just tried your code and after having a look at this part, the deletion message doesn't appear when deleting one file or more than 2 and it should appear.